test/distributed/fsdp/test_fsdp_core.py

```diff
@@ -41,6 +42,23 @@
     )
     sys.exit(0)
 
+params = "cpu_offload,backward_prefetch,sharding_strategy"
+cpu_offload_config = [CPUOffload(offload_params=True), CPUOffload(offload_params=False)]
+backward_prefetch_config = [BackwardPrefetch.BACKWARD_PRE, BackwardPrefetch.BACKWARD_POST, None]
+sharding_strategy_config = [ShardingStrategy.SHARD_GRAD_OP, None]
+configs = list(itertools.product(cpu_offload_config,
+                                 backward_prefetch_config,
+                                 sharding_strategy_config))
+test_name_mapping = {
+    str(CPUOffload(offload_params=True)): "offload_true",
+    str(CPUOffload(offload_params=False)): "offload_false",
+    str(BackwardPrefetch.BACKWARD_PRE): "prefetch_pre",
+    str(BackwardPrefetch.BACKWARD_POST): "prefetch_post",
+    str(ShardingStrategy.SHARD_GRAD_OP): "shard_grad_op",
+}
+
+def subtest_name(*args):
```

Comment:
Nice! I think in follow-up PRs, we can possibly move these lines to `common_fsdp.py` and use this function in the other `test_fsdp_<...>.py` files to unify the approach to having parsable test names.

I can assign this for myself to do later.

Reply:
sounds good, thank you